packages/playwright/src/pages/Arts.ts

```diff
@@ -0,0 +1,63 @@
+import { type Locator, type Page } from '@playwright/test'
+
+export class ArtsPage {
+  readonly page: Page
+  readonly uploadArtsButton: Locator
+  readonly warning: Locator
+  readonly titleInput: Locator
+  readonly uploadButton: Locator
+  readonly descriptionInput: Locator
+  readonly errorMessage1: Locator
+  readonly errorMessage2: Locator
```

Comment:
These variable names could be more descriptive:

```suggestion
  readonly titleErrorMessage: Locator
  readonly descriptionErrorMessage: Locator
```